Tests/ASXCTExtensions.h

```diff
@@ -1,36 +0,0 @@
-/**
- * XCTest extensions for CGGeometry.
- *
- * Prefer these to XCTAssert(CGRectEqualToRect(...)) because you get output
- * that tells you what went wrong.
- * Could use NSValue, but using strings makes the description messages shorter.
- */
-
-#import <XCTest/XCTestAssertionsImpl.h>
-
-#define ASXCTAssertEqualSizes(s0, s1, ...) \
```

Comment:
File was duplicated before. Now it, and all other Common test files, are referenced by the Subspec workspaces but stored in a common place.